### PR TITLE
koordlet: fix kubelet policy nil

### PR DESCRIPTION
**Pull request by huiwq1990**
### V. Checklist

- [x] I have written necessary docs and comments
- [x] I have added necessary unit tests and integration tests
- [x] All checks passed in `make test`

### CODE REVIEW

File: pkg/koordlet/resmanager/cpu_suppress.go [hunk 1]
```diff
@@ -161,6 +161,7 @@ func (r *CPUSuppress) applyBESuppressCPUSet(beCPUSet []int32, oldCPUSet []int32)
 	kubeletPolicy, err := apiext.GetKubeletCPUManagerPolicy(nodeTopo.Annotations)
 	if err != nil {
 		klog.Warningf("failed to get kubelet cpu manager policy, error %v", err)
+		return err
```
Comment (jasonliu747) [inline pkg/koordlet/resmanager/cpu_suppress.go:164]: 可以按照下面的格式返回error
image

Comment (hormes) [reply pkg/koordlet/resmanager/cpu_suppress.go:164]: 不能 return nil

Comment (jasonliu747) [reply pkg/koordlet/resmanager/cpu_suppress.go:164]: 嗯，我的意思是把这两句
```
klog.Warningf("failed to get kubelet cpu manager policy, error %v", err)
return err
```
和下面一样，合并成一句
```
return fmt.Errorf("failed to get kubelet cpu manager policy, error %v", err")
```
这样能够保持日志风格的一致

**Review by jasonliu747 — COMMENTED** [review]
/lgtm

**Pull request merged**